app.py

```diff
@@ -0,0 +1,21 @@
+from flask import Flask
+import redis
+import os
+import socket
+app = Flask(__name__)
+
+@app.route("/")
+def hello():
+
+	r = redis.StrictRedis(host=os.environ.get('REDIS_PORT_6379_TCP_ADDR'), 
```

Comment:
here, you can use `host="redis"` (and no `port`) so it uses DNS instead of env vars. It will work when redis is restarted, and when we have dynamic links.